cypress/support/step-definitions/textbox-steps.js

```diff
@@ -32,22 +105,92 @@ Then('Textbox component is not readOnly', () => {
     .and('have.css', 'background', 'rgb(255, 255, 255) none repeat scroll 0% 0% / auto padding-box border-box');
 });
 
+Then('{string} textbox component is not readOnly', (position) => {
+  switch (position) {
+    case 'First':
+      textbox(FIRST_ELEMENT).should('not.have.css', 'background-color', 'rgba(0, 0, 0, 0)')
+        .and('not.have.css', 'border-bottom-color', 'rgba(0, 0, 0, 0)')
+        .and('not.have.css', 'border-left-color', 'rgba(0, 0, 0, 0)')
+        .and('not.have.css', 'border-right-color', 'rgba(0, 0, 0, 0)')
+        .and('not.have.css', 'border-top-color', 'rgba(0, 0, 0, 0)')
+        .and('have.css', 'background-color', 'rgb(255, 255, 255)');
+      break;
+    case 'Second':
+      textbox(SECOND_ELEMENT).should('not.have.css', 'background-color', 'rgba(0, 0, 0, 0)')
+        .and('not.have.css', 'border-bottom-color', 'rgba(0, 0, 0, 0)')
+        .and('not.have.css', 'border-left-color', 'rgba(0, 0, 0, 0)')
+        .and('not.have.css', 'border-right-color', 'rgba(0, 0, 0, 0)')
+        .and('not.have.css', 'border-top-color', 'rgba(0, 0, 0, 0)')
+        .and('have.css', 'background-color', 'rgb(255, 255, 255)');
+      break;
+    default: throw new Error('There are only two textbox elements on the page');
+  }
+});
+
 Then('Textbox component is labelInline', () => {
   label().should('have.css', TEXT_ALIGN, 'left');
 });
 
+Then('{string} textbox component is labelInline', (position) => {
+  switch (position) {
+    case 'First':
+      labelByPosition(FIRST_ELEMENT).should('have.css', TEXT_ALIGN, 'left');
```

Comment:
could we do not use switch case here ? instead we could just put those two:
`labelByPosition(FIRST_ELEMENT).should('have.css', TEXT_ALIGN, 'left');`
and 
`labelByPosition(SECOND_ELEMENT).should('have.css', TEXT_ALIGN, 'left');`
as one step because we could set align for both elements

Reply:
fixed